cfpb/consumerfinance.gov
# Bugfix: Summary component—bail out of initialization of component when content is too brief; refresh max-height on resize; hide overlay at desktop

## Changes

- Bail out of Summary component initialization when the amount of content is less than the default summary content height.
- Re-calculate max-height when resizing window.
- Hide gradient overlay when it is not in use.

## How to test this PR

1. Pull branch and  `gulp clean && gulp build`
2. Visit http://localhost:8000/ask-cfpb/how-can-i-tell-if-a-friend-neighbor-or-family-member-is-a-victim-of-financial-exploitation-en-1933/ and see that there is a Summary component on mobile.
2. Visit http://localhost:8000/ask-cfpb/can-my-bankcredit-union-charge-me-a-fee-to-use-another-bank-or-credit-unions-atm-en-1081/ and see there is no Summary component on mobile.
3. Visit http://localhost:8000/ask-cfpb/what-is-a-credit-reporting-company-en-1251/ and see that you can click the "Go to our browsable list..." links. Resize page to narrow mobile size and see that content isn't cut off when summary is expanded.
